libass/ass_render.c

```diff
@@ -2263,6 +2271,12 @@ static void render_and_combine_glyphs(ASS_Renderer *render_priv,
             }
             last_info = info;
 
+            ASS_Vector pos, pos_o;
+            info->pos.x = double_to_d6(device_x + d6_to_double(info->pos.x) * render_priv->font_scale_x);
+            info->pos.y = double_to_d6(device_y) + info->pos.y;
+            get_bitmap_glyph(render_priv, info, &pos, &pos_o,
+                             &offset, !current_info->bitmap_count, flags);
+
```

Comment:
Any particular reason for moving this code?

Reply:
It's to get access to `current_info->bitmap_count`. `get_bitmap_glyph` needs to know if that's the first bitmap in run.